content/en/docs/reference/glossary/resourceclaim.md

```diff
@@ -0,0 +1,24 @@
+---
+title: ResourceClaim
+id: resourceclaim
+date: 2025-05-26
+full_link: /docs/reference/kubernetes-api/workload-resources/resource-claim-v1beta1/
```

Comment:
I think all of the v1beta1 links are dead: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/resource-claim-v1beta1/

```suggestion
full_link: /docs/reference/kubernetes-api/workload-resources/resource-claim-v1beta2/
```

https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/resource-claim-v1beta2/